src/network/core/tcp_coordinator.h

```diff
@@ -41,6 +41,7 @@ enum PacketCoordinatorType {
 	PACKET_COORDINATOR_GC_STUN_REQUEST,       ///< Game Coordinator tells client/server to initiate a STUN request.
 	PACKET_COORDINATOR_SERCLI_STUN_RESULT,    ///< Client/server informs the Game Coordinator of the result of the STUN request.
 	PACKET_COORDINATOR_GC_STUN_CONNECT,       ///< Game Coordinator tells client/server to connect() reusing the STUN local address.
+	PACKET_COORDINATOR_GC_NEWGRF_LOOKUP,      ///< Game Coordinator tells the client about the known NewGRFs.
```

Comment:
`known NewGRFs` is a bit vague here. Maybe reference that it is the lookup table for `GC_LISTING`?